src/httprpc.cpp

```diff
@@ -276,8 +274,10 @@ static bool InitRPCAuthentication()
         std::set<std::string>& whitelist = g_rpc_whitelist[strUser];
         if (pos != std::string::npos) {
             std::string strWhitelist = strRPCWhitelist.substr(pos + 1);
-            std::set<std::string> new_whitelist;
-            boost::split(new_whitelist, strWhitelist, boost::is_any_of(", "));
+            std::vector<std::string> whitelistSplit = SplitString(strWhitelist, ", ");
```

Comment:
```suggestion
            std::vector<std::string> whitelist_split = SplitString(strWhitelist, ", ");
```

nit: for new code snake_case ?